tests/integration_test/isomorphism_problem.py

```diff
@@ -1,29 +1,45 @@
 from algorithms.preprocessing import fix_degrees
 from tests.integration_test.algorithm_options import apply_could_be_isomorphic, apply_remove_twins, apply_tree_algorithm, branching_method, apply_complement
+from algorithms.color_initialization import degree_color_initialization
+
+
+def preprocessing(G: "Graph"):
+    """
+    In this method all pre-processing is done that need to be done only one time. This method is called before any
+    other computations are done.
+    :param G: The graph to be preprocessed
+    :return: Factor the amount of automorphisms need to be multiplied with to get the right result
+    """
+    G_complement = apply_complement(G)
```

Comment:
Misschien kun je dit beter G_preprocessed noemen ofzo, deze naam is misschien een beetje verwarrend alsof het sowieso de complement is. Iets verderop heb ik daar ook nog een opmerking over.